### DISCUSSION TITLE
Morphia Core

### DISCUSSION THREAD

**Discussion by NOTiFYcouk** [MorphiaOrg/morphia#2288] [General]
Where can I download the JAR of 'dev.morphia.morphia:morphia-core:2.4.0-SNAPSHOT' ?

**Comment by evanchooly** [comment]
2.4 is still under development and shouldn't be used in production code but if you want to experiment, instructions on how to add it to your build can be found [here](https://morphia.dev/morphia/2.4/index.html).
